compiler/testSuite/14-bug/infiniteArray-v.bal

```diff
@@ -0,0 +1,7 @@
+import ballerina/io;
+type Arr Arr[];
```

Comment:
Disallowing types like this is a spec change, so I think we should create a separate directory for this - maybe 14-infinite.  The intersection cases are bugs, so they should go in 14-bug.

Reply:
fixed